src/data_rate.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package helium;
+
+enum date_rate {
+    SF12BW125 = 0;
+    SF11BW125 = 1;
+    SF10BW125 = 2;
+    SF9BW125 = 3;
+    SF8BW125 = 4;
+    SF7BW125 = 5;
+    SF12BW250 = 6;
+    SF11BW250 = 7;
+    SF10BW250 = 8;
+    SF9BW250 = 9;
+    SF8BW250 = 10;
+    SF7BW250 = 11;
+    SF12BW500 = 12;
+    SF11BW500 = 13;
+    SF10BW500 = 14;
+    SF9BW500 = 15;
+    SF8BW500 = 16;
+    SF7BW500 = 17;
+    LRFHSS1BW137 = 18;
+    LRFHSS2BW137 = 19;
+    LRFHSS1BW336 = 20;
+    LRFHSS2BW336 = 21;
+    LRFHSS1BW1523 = 22;
+    LRFHSS2BW1523 = 23;
+    FSK50 = 24;
+    RFU = 25;
```

Comment:
fRFU is "reserved for future use", and Michael asked that this be assigned to the 0th slot, which of course upsets the numbering

```suggestion
    RFU = 0;
```

Reply:
Ah, right, will adjust.

Reply:
not quite how protobuf is intended to be used. Remove RFU.. protobuf messages can be extended for future use and you shouldn't be assigning index